config/toml.go

```diff
@@ -66,600 +68,6 @@ func WriteConfigFile(configFilePath string, config *Config) {
 
 // Note: any changes to the comments/variables/mapstructure
 // must be reflected in the appropriate struct in config/config.go.
-const defaultConfigTemplate = `# This is a TOML config file.
-# For more information, see https://github.com/toml-lang/toml
-
-# NOTE: Any path below can be absolute (e.g. "/var/myawesomeapp/data") or
-# relative to the home directory (e.g. "data"). The home directory is
-# "$HOME/.cometbft" by default, but could be changed via $CMTHOME env variable
-# or --home cmd flag.
-
-# The version of the CometBFT binary that created or
-# last modified the config file. Do not modify this.
-version = "{{ .BaseConfig.Version }}"
-
-#######################################################################
-###                   Main Base Config Options                      ###
-#######################################################################
-
-# TCP or UNIX socket address of the ABCI application,
-# or the name of an ABCI application compiled in with the CometBFT binary
-proxy_app = "{{ .BaseConfig.ProxyApp }}"
-
-# A custom human readable name for this node
-moniker = "{{ .BaseConfig.Moniker }}"
-
-# Database backend: goleveldb | cleveldb | boltdb | rocksdb | badgerdb | pebbledb
```

Comment:
`cleveldb | boltdb`

this has disappeared in the embedded file

Reply:
Hi @sergio-mena, the reason they disapperaed is because `boltdb` and `cleveldb` has been removed in this PR https://github.com/cometbft/cometbft/pull/2786/files#diff-003ce8db64b969b85ae335f805c56b16bed98a637ddfb4ac8da5ca04f6632ed9

Reply:
#2786 is on `main`. Here we are in `v1.x`, so -- if I'm not missing anything obvious -- those should still exist on this branch

Reply:
BTW, are you using git's 3-way merge info in order to fix this kind of conflicts?